interfaces/builtin/audio_playback.go

```diff
@@ -111,6 +119,16 @@ owner /run/user/[0-9]*/ r,
 owner /run/user/[0-9]*/pulse/ rw,
 `
 
+const audioPlaybackPermanentSlotAppArmorCore = `
+# This allows to share screen in Core Desktop
+owner /run/user/[0-9]*/pipewire-[0-9] rwk,
+
+# This allows to read the wireplumber configuration if
+# pipewire runs inside the container
+/etc/pulse/ r,
+/etc/pulse/** r,
```

Comment:
Is it actually wireplumber reading these config files, or just the PulseAudio client libraries? I thought all the wireplumber config was in `/usr/share/wireplumber` (which we currently have access to in the UCD image because we're using the custom base snap)?

I don't see a problem in granting this access, but the comment should more accurately describe what it's for.

Reply:
Fixed. I think that this new explanation is better.

Reply:
Reading the updated description, I'm still a little surprised that wireplumber itself would be making this access.

Reply:
I remember to add it because I saw some deny accesses... but I can test it again.